src/media/css/app-list-filters.styl

```diff
@@ -56,7 +56,12 @@ App listing filters.
             background-color: $white;
         }
         + a {
-            margin-left: 30px;
+            +ltr() {
```

Comment:
Just added this change so there's spacing between "Popular" and "New".

![screenshot 2015-04-23 10 41 32](https://cloud.githubusercontent.com/assets/211578/7300993/54ee7518-e9a5-11e4-8589-195c8fe83186.png)